storage/remote/queue_manager.go

```diff
@@ -170,18 +195,18 @@ func newQueueManagerMetrics(r prometheus.Registerer, rn, e string) *queueManager
 		Help:        "The number of shards that the queues shard calculation wants to run based on the rate of samples in vs. samples out.",
 		ConstLabels: constLabels,
 	})
-	m.bytesSent = prometheus.NewCounter(prometheus.CounterOpts{
+	m.samplesBytesTotal = prometheus.NewCounter(prometheus.CounterOpts{
 		Namespace:   namespace,
 		Subsystem:   subsystem,
-		Name:        "sent_bytes_total",
-		Help:        "The total number of bytes sent by the queue.",
+		Name:        "samples_bytes_total",
+		Help:        "The total number of bytes of samples sent by the queue after compression.",
 		ConstLabels: constLabels,
 	})
-	m.maxSamplesPerSend = prometheus.NewGauge(prometheus.GaugeOpts{
```

Comment:
Do we not want this anymore?

Reply:
I believe this happened because it was added recently and the PR was old, hence the merge thinks we want to remove it. I am keeping `maxSamplesPerSend`.

Reply:
Yeah, that looks like a merge failure.